src/goto-programs/remove_calls_no_body.h

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************\
+
+Module: Remove calls to functions without a body
+
+Author: Daniel Poetzl
+
+\*******************************************************************/
+
+/// \file
+/// Remove calls to functions without a body
+
+#ifndef CPROVER_GOTO_PROGRAMS_REMOVE_CALLS_NO_BODY_H
+#define CPROVER_GOTO_PROGRAMS_REMOVE_CALLS_NO_BODY_H
+
+#include "goto_functions.h"
+
+class remove_calls_no_bodyt
+{
+protected:
+  typedef goto_functionst::goto_functiont goto_functiont;
```

Comment:
I am wondering whether this typedef is of much value - it seems to be used twice, and one of those you could even eliminate given my earlier comment.

Reply:
Is that line a left-over to be removed?